colossalai/tensor/comm_spec.py

```diff
@@ -79,6 +79,132 @@ def _all_reduce(tensor, comm_spec, async_op=False):
             return tensor
 
 
+def _mix_gather(tensor, comm_spec):
```

Comment:
Add more description for this operation.

Reply:
ok